ssl/record/rec_layer_d1.c

```diff
@@ -170,7 +170,7 @@ static void dtls_unbuffer_record(SSL_CONNECTION *s)
  * Return up to 'len' payload bytes received in 'type' records.
  * 'type' is one of the following:
  *
- *   -  SSL3_RT_HANDSHAKE (when ssl3_get_message calls us)
+ *   -  SSL3_RT_HANDSHAKE (when tls_get_message_header calls us)
```

Comment:
In this case I would just delete the text in parens after SSL3_RT_HANDSHAKE. It actually makes no sense any more. This function is called from quite a number of places with the type set to SSL3_RT_HANDSHAKE, and tls_get_message_header isn't one of them (`dtsl1_read_bytes` is only relevant to DTLS, and `tls_get_message_header` is only relevant to TLS).

Reply:
Okay.

Reply:
You still did not revert this change.